recipes/samtools/1.16.1/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "1.16.1" %} # reset build to zero on changing
+{% set htslib_version = "1.16" %}
+
+package:
+  name: samtools-pkg
+  version: "{{ version }}"
+
+about:
+  home: https://github.com/samtools/samtools
+  license: MIT
+  summary: C library for high-throughput sequencing data formats.
+
+build:
+  number: 0
+
+source:
+  git_url: https://github.com/samtools/samtools.git
+  git_rev: 1.16.1
```

Comment:
`{{ version }}` here too? Unless there can be variations in the way a samtools release is tagged ... I don't know if that's the case.

Reply:
There may very well be variations. A 'tag' and a 'version' may easily refer to different things, even if they are usually the same string. In any case, we have no control over tagging since that's done by SAM rather than NPG.
So my inclination is to leave this as is.